packages/manager/core/package.json

```diff
@@ -19,8 +19,8 @@
     "ovh-api-services": "^3.18.0"
   },
   "scripts": {
-    "build": "rollup -c",
-    "dev": "rollup -c --watch",
+    "build": "rollup -c --environment BUILD:production",
```

Comment:
Is there any reason to specify environment variable name in uppercase?
For instance here we are using lowercase: https://github.com/ovh-ux/manager/blob/develop/packages/manager/layout-ovh/package.json#L8